### ISSUE TITLE
IE9: Unable to call apply on console.error when encountering an unknown tag

### ISSUE THREAD

**Issue by nhunzaker** [facebook/react#13610]
This was introduced in React 16.5.0.

If IE9 encounters an unknown html element in development, React crashes as it attempts to call console.error.apply here:

https://github.com/facebook/react/blob/b87aabdfe1b7461e7331abb3601d9e6bb27544bc/packages/shared/warningWithoutStack.js#L30

This is reproduceable with the following CodePen:

**Code**
https://codepen.io/nhunzaker/pen/YOeZXv

**Demo**
https://s.codepen.io/nhunzaker/debug/YOeZXv/mWMoNxzPjbZk

**Comment by gaearon** [comment]
What's the error?

**Comment by nhunzaker** [comment]
The spread operator turns into apply like so:

```javascript
// Before: console.error('Warning: ' + format, ...stringArgs);
console.error.apply(console, ['Warning' + format].concat(stringArgs))
```

IE9 doesn't support `apply` on `console.error`:

<img width="666" alt="screen shot 2018-09-10 at 1 50 50 pm" src="https://user-images.githubusercontent.com/590904/45323618-8f15d400-b500-11e8-83b6-2adb1b104fef.png">

**Comment by link-alex** [comment]
I can try to check this one as "good first issue" if you don't mind :)

**Comment by gaearon** [comment]
Should be fixed in 16.5.1
